tests/js/sentry-test/reactTestingLibrary.tsx

```diff
@@ -36,17 +29,13 @@ function createProvider(contextDefs: Record<string, any>) {
   };
 }
 
-function makeAllTheProviders({context, organization}: ProviderOptions) {
+function makeBaseProviders(context?: Record<string, any>) {
   const ContextProvider = context ? createProvider(context) : Fragment;
   return function ({children}: {children?: React.ReactNode}) {
     return (
       <ContextProvider>
         <CacheProvider value={cache}>
-          <ThemeProvider theme={lightTheme}>
-            <OrganizationContext.Provider value={organization ?? null}>
-              {children}
-            </OrganizationContext.Provider>
-          </ThemeProvider>
+          <ThemeProvider theme={lightTheme}>{children}</ThemeProvider>
```

Comment:
in the future, we might want to test a dark theme as well, so I was wondering if we go with this option, if we want to remove ThemeProvider from here as well. The downside is that we will have to wrap all our components with
`<ThemeProvider theme={...}>`